canonical-web-and-design/ubuntu.com
# Vendor pages using partner data

## Done

- Replaced static vendors_data with data from https://partners.ubuntu.com endpoint
- Because partners data company/make/vendor names don't match with certification API vendor names, a way to do this was to map them to the corresponding certification API name.
- Using partners data allows us to pull any vendor info without manually adding it, but because of previously mentioned inconsistency, we may need to map them manually sometimes.
- For UX consistency, certification API vendor names has been kept for the titles. E.g. AAEON Technology Inc. instead of AAEON
- Added python test for vendor pages

## QA

- Visit some old vendor pages to check that they work as before and have the header strip content  https://ubuntu-com-11173.demos.haus/certified/vendors/Dell (compared with https://ubuntu-com-11173.demos.haus/certified/vendors/Dell)
- Visit some new vendor header strip content, added using the partners data such as  https://ubuntu-com-11173.demos.haus/certified/vendors/AAEON%20Technology%20Inc. (doesn't exist currently /certified/vendors/AAEON%20Technology%20Inc.)

Let me know if any vendors are missing. Otherwise, if data is missing most likely we just don't have that vendor information in the partners data.

## Issue / Card

Fixes canonical-web-and-design/web-squad#4699

## Screenshots

![aaeon-vendor](https://user-images.githubusercontent.com/14939793/150520802-216dbc7f-8701-47f1-bb16-d174ade0043d.png)


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
